### PR TITLE
[4.0]namespace administrator components com_cpanel

### PR DESCRIPTION
**Pull request by brianteeman**
continuing the work to namespace joomla4 one admin component at a time.

please wait for drone to successfully complete before reviewing.

### CODE REVIEW

File: administrator/components/com_csp/Controller/DisplayController.php [hunk 1]
```diff
@@ -49,11 +49,11 @@ public function display($cachable = false, $urlparams = false)
 			$link = HTMLHelper::_(
 				'link',
 				'#plugin' . $httpHeadersId . 'Modal',
-				\JText::_('COM_CSP_SYSTEM_PLUGIN'),
+				Text::_('COM_CSP_SYSTEM_PLUGIN'),
```
Comment (Quy) [inline administrator/components/com_csp/Controller/DisplayController.php:53]: Add `use Joomla\CMS\Language\Text;` to this file.

File: administrator/components/com_csp/View/Reports/HtmlView.php [hunk 2]
```diff
@@ -116,7 +116,7 @@ protected function addToolbar()
 	{
 		$canDo = ContentHelper::getActions('com_csp');
 
-		ToolbarHelper::title(\JText::_('COM_CSP_REPORTS'), 'generic');
+		ToolbarHelper::title(Text::_('COM_CSP_REPORTS'), 'generic');
```
Comment (Quy) [inline administrator/components/com_csp/View/Reports/HtmlView.php:120]: Add `use Joomla\CMS\Language\Text;` to this file.

File: administrator/components/com_csp/Controller/DisplayController.php (hunk 1 above)
Comment (brianteeman) [reply administrator/components/com_csp/Controller/DisplayController.php:53]: fixed

File: administrator/components/com_csp/View/Reports/HtmlView.php (hunk 2 above)
Comment (brianteeman) [reply administrator/components/com_csp/View/Reports/HtmlView.php:120]: fixed

**Comment by wilsonge** [comment]
Thanks guys!

**Comment by brianteeman** [comment]
Thanks

**Pull request merged**